packages/riipen-ui/src/components/AccordionContext.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const AccordionContext = React.createContext({});
+
+if (process.env.NODE_ENV !== "production") {
+  AccordionContext.displayName = "AccordionContext";
+}
```

Comment:
do we need to manually do this? if we are minifying i think it should get removed automatically

Reply:
true